Assets/Scripts/MovePB.cs

```diff
@@ -24,18 +25,22 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        _playerInput = Input.GetAxis("Vertical");
-        _rotationInput = Input.GetAxis("Horizontal");
+        _playerVerticalInput = Input.GetAxis("Vertical");
+        _playerHorizontalInput = Input.GetAxis("Horizontal");
         _userJumped = Input.GetButton("Jump");
     }
 
     private void FixedUpdate()
     {
         _userRot = _transform.rotation.eulerAngles;
-        _userRot += new Vector3(0, _rotationInput, 0);
-
-        _transform.rotation = Quaternion.Euler(_userRot);
-        _rigidbody.velocity += transform.forward * _playerInput * _inputScale;
+        _userRot += new Vector3(0, _playerHorizontalInput, 0);
+
```

Comment:
If we do this, then players can only move on 90 degree angles. Is that what we want?

Reply:
I wasn't really sure on what we want. I can change that, I will probably change the player's character from a sphere to something that shows its orientation. Then use "DW" to rotate the character without rotating the camera and move around.